README.md

```diff
@@ -26,7 +26,10 @@ check out and contribute to this repository.
 Important links:
 
 - [Workshops](workshops/README.md) :: Workshops for use in clubs
-- https://slack.hackclub.io :: Join our [Slack](https://slack.com/) community
+- https://slack.hackclub.io :: Join
+  [our Slack](https://starthackclub.slack.com/) community
```

Comment:
@zachlatta Rather than linking to _slack_ the company, I thought it was more relevant to link to our slackin.